<ui version="4.0" >
 <class>XWidgetEditFunction</class>
 <widget class="QWidget" name="XWidgetEditFunction" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Function Information</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Function Name</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QLineEdit" name="name" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>States</string>
         </property>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Current</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="currentState" />
          </item>
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Next</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nextState" />
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Function Description</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QsciScintilla" name="description" >
            <property name="font" >
             <font>
              <family>Monospace</family>
              <pointsize>10</pointsize>
              <stylestrategy>PreferDefault</stylestrategy>
             </font>
            </property>
            <property name="toolTip" >
             <string/>
            </property>
            <property name="whatsThis" >
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>Condition</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QTextEdit" name="condition" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Code Implementation</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QTextEdit" name="code" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QsciScintilla</class>
   <extends>QFrame</extends>
   <header>Qsci/qsciscintilla.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
